spectrometer.cabal

```diff
@@ -46,12 +45,13 @@ common lang
     RankNTypes
     ScopedTypeVariables
     StandaloneDeriving
-    StandaloneKindSignatures
     StrictData
     TupleSections
     TypeApplications
     TypeOperators
     TypeSynonymInstances
+    ImportQualifiedPost
+    StandaloneKindSignatures
```

Comment:
[nit]: did cabal-fmt put these out-of-order?

Reply:
yeah, I'm not sure why it's doing that 😕